docs/DeveloperGuide.adoc

```diff
@@ -297,33 +297,47 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new patient/donor/doctor |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |delete a patient/donor/doctor |remove entries that I no longer need
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |find a patient/donor/doctor using certain keywords |locate details of persons without having to go through the entire list
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |user |find potential donors for a patient |help the patient find a donor as fast as possible
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`* * *` |user |process a patient and a donor |prepare them for the surgery process
+
+|`* * *` |user |see list of a particular patient's potential donors|know list of potential donors for a particular patient
+
+|`* * *` |user |see list of all patients/donors/doctors|
+
+|`* * *` |careless user |edit a patient/donor/doctor|
+
+|`* * *` |user |mark a match as done|know that a particular match has undergone a surgery
+
+|`* * *` |user |sort the potential donors by success rate/priority|match a patient with the best possible donor
+
+|`* * ` |user |add multiple data using external file|reduce the time to add in a data into the system
```

Comment:
"Multiple Data" and "external file" are vague terms, perhaps consider specifying what kind of data and what format it should be in?